changelog.txt

```diff
@@ -14,6 +14,135 @@ Old-style release channel:
+  [SF-2400] - SensorFu Beacon: SensorFu Beacon output events do not work
+  [SF-2398] - TrafficGuard Manager: Fix autovacuum and autoanalyze for tgmanager
+  [SF-2395] - FleetCert: python3.8 missing from sensor in some installations
+  [SF-2394] - PortDiff Instrument: Banner diff event does not contain all removed banenrs
+  [SF-2375] - Sensor Orchestrator: Sensor orchestrator network thread crash while upgrading
+  [SF-2360] - UI: Instrument custom config UI boolean values are wrong
+  [SF-2353] - API: Subscription worker _send_worker() is not cancelled properly on all cases
+  [SF-2348] - SensorFu Beacon: Beacon instrument does not handle restarting (or failure to stop) non-running beacon binary
+  [SF-2339] - Sensor Orchestrator: Orchestrator should disable unneeded features to bridged veth legs
+  [SF-2335] - Log Forwarder Instrument: Force a timeout for beats connection
+  [SF-2334] - Log Forwarder Instrument: logstash force kill is broken
+  [SF-2327] - PortDiff Instrument: PortDiff does not handle shutdown when scan is running
+  [SF-2317] - Rule Importer: Rule Importer Instrument refuses to start
+  [SF-2307] - Sensor Orchestrator: Invalid network configuration breaks Sensor Orchestrator logic
+  [SF-2306] - Replay Instrument, UI: Replay Instrument displays duplicate errors
+  [SF-2300] - API: Mock sensor's add_event() does not handle dots in keys
+  [SF-2291] - Rule Manager: Upgrading from 2.1.4/2.2.3 to 2.3.X may result in unusable database
+  [SF-2290] - UI: Sensor UI: Create users config if it's missing
+  [SF-2282] - UI: DHCP checkbox disappears from UI and never comes back
+  [SF-2281] - Replay Instrument: Replay instrument refuses to start task with the same filename
+  [SF-2279] - Base Packages, Sensor Image: Apt may refuse to install sensorfleet packages when grub-efi-amd64-signed is installed
+  [SF-2277] - Rule Manager: Implement proper error handling and/or restarting of download poller task
+  [SF-2270] - UI: Cannot view events from another sensor
+  [SF-2255] - PassiveDNS: PassiveDNS database slowdown due to missing periodic autoanalyze
+  [SF-2237] - HelloWorld Instrument: Helloworld example shutdown/main logic is broken
+  [SF-2235] - UI: FM UI takes a long time to restart/stop
+  [SF-2227] - Rule Manager: Rule Manager import regex is insufficient for some rule sources
+  [SF-2221] - Recorder Instrument: Recorder fetch PCAP API does not return all packets with the requested filter
+  [SF-2197] - Instruments should be started on 'default' runlevel
+  [SF-2145] - UI: UI error after mis-selecting bridge
+  [SF-1747] - Sensor Orchestrator: sensor-orchestrator removes default route
+  [SF-1122] - Recorder Instrument: moloch-capture does not stop without kill -9
+  [SF-1008] - UI: Automatic IP assignment for bridges is broken
+  [SF-978] - UI: Version query fails after update
+  [SF-515] - UI: New context menu should close others
+
+- New features and other improvements:
+  [SF-2430] - UI: Force password change backend
```

Comment:
Not yet merged even if task is done. Better to postpone and add to the changelog when whole feature is released.